src/Craned/Craned/Craned.cpp

```
@@ -81,111 +81,97 @@
-        g_config.CranedLogFile = g_config.CraneBaseDir + kDefaultCranedLogPath;
+        g_config.CranedLogFile =
+            g_config.CraneBaseDir /
+            value_or(config["CranedLogFile"], kDefaultCranedLogPath);
 
       if (parsed_args.count("debug-level"))
         g_config.CranedDebugLevel =
             parsed_args["debug-level"].as<std::string>();
-      else if (config["CranedDebugLevel"])
-        g_config.CranedDebugLevel =
-            config["CranedDebugLevel"].as<std::string>();
       else
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)